submissions/madmaxWMFU/Friends-App/js/app.js

```diff
@@ -0,0 +1,100 @@
+const allFriends = {
+    currentList: [],
+    changeList: [],
+    countFriends: 52,
+    minAge: 0,
+    maxAge: 150
+};
+
+const friendZone = document.querySelector(".friends-zone");
+const filters = document.querySelector(".filters-input");
+const sorts = document.querySelector(".search-sort-options");
+const resetFilters = document.querySelector(".reset-button");
+
+const getDataApi = async () => {
+    const response = await fetch(`https://randomuser.me/api/?results=${allFriends.countFriends}`);
+    const data = await response.json();
+    allFriends.currentList = data.results;
```

Comment:
This is a side-effect which should be avoided. For now it is ok, because you are newbie, but in general, you should try to avoid such things.

As as example, this function must have one responsibility and should only return data, not render them or save them elsewhere. All this things must happen outside by it's parent. 

We had materials about purity and single-responsibility, so you should already know this topics. But fill free to ask here anything regarding this topic.